src/v2/view-builder/components/AuthenticatorVerifyOptions.js

```diff
@@ -19,7 +19,12 @@ const AuthenticatorRow = View.extend({
           </div>\
         </div>\
         <div class="authenticator-description">\
```

Comment:
It wouldn't work because `.authenticator-description` is used for `display: flex;` (The button needs to be on the right side) https://github.com/okta/okta-signin-widget/blob/4.2/assets/sass/v2/_select-authenticator.scss#L43

Reply:
Also, I don't want to change the line-height unless there's a description && it is not in enroll.

Reply:
how about refactoring the class name a little bit (avoid sublabel)
`has-description` only show up when having `description`

```
<div class="authenticator-label-description-container has-description">
  <div class="authenticator-label-description-container-inner">
    h3 {{label}}
    p {{description}}
  </div>
</div>
```

Reply:
I'm thinking this would be cleaner, I can target the element directly - 
```
        <div class="authenticator-description">\
          <div>\
            <h3 class="authenticator-label {{#if description}}authenticator-label--small{{/if}}">{{label}}</h3>\
            {{#if description}}\
              <p class="authenticator-description--text">{{description}}</p>\
            {{/if}}\
          </div>\
          <div class="authenticator-button"></div>\
        </div>\
```
Then I'll change the CSS accordingly